snapcraft/internal/parser.py

```diff
@@ -134,7 +134,23 @@ def _update_source(part, origin):
     return part
 
 
-def _process_entry_parts(entry_parts, parts, origin, maintainer, description):
+def _replace_variables(item, replacements):
```

Comment:
Hmm, there might have been a miscomm somewhere :)

The general idea was to remove the `_` from https://github.com/snapcore/snapcraft/blob/master/snapcraft/internal/project_loader.py#L201 and adapt that ;-)